assets/aml-benchmark/components/src/aml_benchmark/inference_postprocessor/inference_postprocessor.py

```diff
@@ -175,10 +179,16 @@ def read_ground_truth_dataset(self) -> pd.DataFrame:
                 result_df[self.ground_truth_column_name] = actual_df[
                     self.ground_truth_column_name
                 ]
+            if self.additional_columns:
+                for column in self.additional_columns.split(","):
+                    if column in actual_df.columns:
```

Comment:
Same here-
1. We can trim column for whitespaces at start and end.
2. Ignore if it empty.
3. Throw user error if column is not in actual_df.